FirebasePerformance/Sources/Instrumentation/Network/Delegates/FPRNSURLSessionDelegate.m

```diff
@@ -39,12 +39,8 @@ - (void)URLSession:(NSURLSession *)session
   @try {
     FPRNetworkTrace *trace = [FPRNetworkTrace networkTraceFromObject:task];
     trace.requestSize = totalBytesSent;
-    if (totalBytesSent >= totalBytesExpectedToSend) {
-      if ([task.response isKindOfClass:[NSHTTPURLResponse class]]) {
-        NSHTTPURLResponse *response = (NSHTTPURLResponse *)task.response;
-        [trace didCompleteRequestWithResponse:response error:task.error];
-        [FPRNetworkTrace removeNetworkTraceFromObject:task];
-      }
+    if (totalBytesSent == totalBytesExpectedToSend) {
```

Comment:
can we leave this at ">=" instead of "=="? I remember usecases where the bytes sent was higher than expected and it might help in those usecases. Same in other places as well.

Reply:
Updated!